toppra_trajectory:
  velocities: [1.3, 1.3, 1.3]
  accelerations: [2.0, 2.0, 2.0]
  sampling_frequency: 100.0